README.md

```diff
@@ -10,4 +10,449 @@ TODO: description
+- `400` If the first_name, last_name, or phone_number is empty or a stream of empty spaces
+- `404` if the medicalContactId is invalid
+
+#### `DELETE /api/medical-contacts/:medicalContactId?` - Deactivates a medical contact
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `401` if the user is not logged in
+- `403` if the user is not the owner of the medical contacts
+- `404` if the medicalContactId is invalid
+- `409` if the medicalContactId is already deactivated 
+
+### Medication 
+
+#### `GET /api/medications` - Get a user's medications
+
+**Returns**
+
+- An array of medications created by the user in alphabetical order with respect to the name of the medication
+
+**Throws**
+
+- `401` if the user is not logged in 
+
+#### `POST /api/medications` - Create a new medication
+
+**Body**
+
+- `name` _{string}_ - The medication's name
+- `generic_name` _{string}_ - The medication's generic compound name
+- `dose` _{string}_ - The medication's dose
+- `notes` _{string}_ - Notes about the medication
+
+**Returns**
+
+- A success message
+- A object with the created medication
+
+**Throws**
+
+- `401` if the user is not logged in
+- `400` If the name, generic_name, or dose is empty or a stream of empty spaces
+
+#### `PATCH /api/medications/:medicationId?` - Edit the information of a medication
+
+**Body**
+
+- `name` _{string}_ - The medication's name (optional)
+- `generic_name` _{string}_ - The medication's generic compound name (optional)
+- `dose` _{string}_ - The medication's dose (optional)
+- `notes` _{string}_ - Notes about the medication (optional)
+
+**Returns**
+
+- A success message
+- A object with the edited medication
+
+**Throws**
+
+- `401` if the user is not logged in
+- `403` if the user is not the owner of the medication
+- `400` If the name, generic_name, or dose is empty or a stream of empty spaces
+- `404` if the medicationId is invalid
+
+#### `DELETE /api/medications/:medicationId?` - Deactivates a medication
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `401` if the user is not logged in
+- `403` if the user is not the owner of the medication
+- `404` if the medicationId is invalid
+- `409` if the medicationId is already deactivated 
+
+### Entry
+
+#### `GET /api/entries` - Get all entries of the user
+
+**Returns**
+- The array of entries by the user sorted in descending order by date
+
+**Throws** 
+- `401` if the user is not logged in
+
+#### `POST /api/entries` - Make a new entry in the log
+
+**Body**
+- `type` _{string}_ - The type of the log entry
+- `detail` _{string}_ - The detail of the log entry
+- `notes` _{string}_ - The log entry's notes (optional)
+- `date` _{Date}_ - The date associated with the log entry (optional)
+
+**Returns**
+- A success message
+- The newly created entry object
+
+**Throws** 
+- `401` if the user is not logged in
+- `400` if the `type` or `detail` is empty or a stream of empty spaces
+
+#### `PATCH /api/entries/:entryId?` - Editing an entry in the log
```

Comment:
did we decide question marks or no question marks? Seems like we are half and half right now.